### PR TITLE
Poly 23

### PR DESCRIPTION
**Pull request by Iangyl**

### CODE REVIEW

File: src/common/components/LocalizationPicker/LocalizationPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    },
+    [setLanguage, translation.Language],
+  );
+
+  const getLanguage = useMemo(() => {
+    switch (language) {
+      case 'en': return 'English'
+      default: return 'English'
+    }
+  }, [language])
```
Comment (DmitriyIvchenko) [inline src/common/components/LocalizationPicker/LocalizationPicker.tsx:36]: please Add ENUM mapper for lang names

Comment (DmitriyIvchenko) [reply src/common/components/LocalizationPicker/LocalizationPicker.tsx:36]: languages[languageCode].name

File: src/common/components/LocalizationPickerInfo/LocalizationPickerInfo.module.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.light {
+  --color: var(--color_blue_gray);
+  --bg_dropdown: var(--color_white);
+}
+
+.common_header_gas_info_menu {
+  --font_size: 18px;
+  --font_weight_bold: 800;
```
Comment (DmitriyIvchenko) [inline src/common/components/LocalizationPickerInfo/LocalizationPickerInfo.module.scss:15]: exists in variables already

File: src/common/components/LocalizationPickerInfo/LocalizationPickerInfo.module.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  --bg_dropdown: var(--color_blue_gray_bg);
+}
+
+.light {
+  --color: var(--color_blue_gray);
+  --bg_dropdown: var(--color_white);
+}
+
+.common_header_gas_info_menu {
+  --font_size: 18px;
```
Comment (DmitriyIvchenko) [inline src/common/components/LocalizationPickerInfo/LocalizationPickerInfo.module.scss:14]: also this

File: src/common/localization/translations/en.ts [hunk 4]
```diff
@@ -63,6 +63,10 @@ export default {
     inr: 'INR',
     inrSecondary: 'Indian Rupee',
   },
+  Language: {
+    en: 'EN',
+    enSecondary: 'English'
```
Comment (DmitriyIvchenko) [inline src/common/localization/translations/en.ts:68]: EN? what about other languages???

File: src/pages/SettingsPage/SettingsBody/SettingsBody.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            <div className={style.main_addr}>
+              <ProfileWallet address={user} size={6} />
+            </div>
+            <ProfileWallet address={user} size={6} />
+          </div>
+        </div>
+      </div>
+      <div className={style.settings_prop}>
+        <div className={style.row}>
+          <div className={style.left}>Theme</div>
```
Comment (DmitriyIvchenko) [inline src/pages/SettingsPage/SettingsBody/SettingsBody.tsx:38]: translations

File: src/pages/SettingsPage/SettingsBody/SettingsBody.tsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+          <div className={style.right}><ThemeSwitcher /></div>
+        </div>
+        <div className={style.row}>
+          <div className={style.left}>Base Currency</div>
```
Comment (DmitriyIvchenko) [inline src/pages/SettingsPage/SettingsBody/SettingsBody.tsx:42]: translations

File: src/pages/SettingsPage/SettingsBody/SettingsBody.tsx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+          <div className={style.right}><CurrencyPicker /></div>
+        </div>
+        <div className={style.row}>
+          <div className={style.left}>Language</div>
```
Comment (DmitriyIvchenko) [inline src/pages/SettingsPage/SettingsBody/SettingsBody.tsx:46]: translations

File: src/pages/SettingsPage/SettingsBody/SettingsBody.tsx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+import LocalizationPicker from 'common/components/LocalizationPicker/LocalizationPicker';
+
+function SettingsBody() {
+  const theme = useTheme()
+  const { user } = useAuth()
+  const translation = useTranslation()
+
+  return (
+    <div className={classNames(style[theme])}>
+      <div className={style.head_title}>Settings</div>
```
Comment (DmitriyIvchenko) [inline src/pages/SettingsPage/SettingsBody/SettingsBody.tsx:23]: translations

File: src/pages/SettingsPage/SettingsBody/SettingsBody.tsx [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+      <div>
+        <div className={style.profile_sign}>Profile</div>
```
Comment (DmitriyIvchenko) [inline src/pages/SettingsPage/SettingsBody/SettingsBody.tsx:25]: translations

File: src/pages/SettingsPage/SettingsBody/SettingsBody.module.scss [hunk 10]
```diff
@@ -0,0 +1,68 @@
+@import 'common/themes/mixins';
```
Comment (DmitriyIvchenko) [inline src/pages/SettingsPage/SettingsBody/SettingsBody.module.scss:1]: theming works for this page?

File: src/common/components/LocalizationPicker/LocalizationPicker.tsx (hunk 1 above)
Comment (DmitriyIvchenko) [reply src/common/components/LocalizationPicker/LocalizationPicker.tsx:36]: languages {en: {name: English, code: en}} etc

**Review by DmitriyIvchenko — CHANGES_REQUESTED** [review]

File: src/common/localization/translations/en.ts (hunk 4 above)
Comment (Iangyl) [reply src/common/localization/translations/en.ts:68]: I don't know what languages would be provided. In config localization file only "en" language exist.

**Review by DmitriyIvchenkoYozmaTech — APPROVED** [review]

**Pull request merged**